charts/mlrun-ce/templates/config/mlrun-env-configmap.yaml

```diff
@@ -1,17 +1,22 @@
 {{ if .Values.mlrun.enabled}}
+{{- $bucket_name := .Values.global.infrastructure.aws.bucketName | default "mlrun" }}
```

Comment:
u can define that under helpers.tpl and reuse from many places

Reply:
Fixed